### PR TITLE
Fix the API key request form UI and make it consistent with rest of UI

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

**Comment by TG1999** [comment]
Screenshot from 2022-11-15 20-33-45

File: vulnerabilities/forms.py [hunk 1]
```diff
@@ -48,7 +48,22 @@ class Meta:
...
+        email_field.widget.attrs["style"] = "width: 50%"
+        email_field.widget.attrs["placeholder"] = "e.g. foo@bar.com"
+        first_name_field.label = "First Name"
+        first_name_field.widget.attrs["class"] = "input"
+        first_name_field.widget.attrs["style"] = "width: 50%"
+        first_name_field.widget.attrs["placeholder"] = "e.g. Jon"
+        last_name_field.label = "Last Name"
+        last_name_field.widget.attrs["class"] = "input"
+        last_name_field.widget.attrs["style"] = "width: 50%"
+        last_name_field.widget.attrs["placeholder"] = "e.g. Doe"
```
Comment (tdruez) [inline vulnerabilities/forms.py:66]: "e.g. " are not needed.

File: vulnerabilities/templates/api_user_creation_form.html [hunk 2]
```diff
@@ -11,15 +11,33 @@
...
+        <div class="message-body">
+          {{ errors }}
+        </div>
+        {% endfor %}
+      </div>
+    <h2 class="subtitle mb-0 pt-2 mb-2">
+      <b>VulnerableCode API key request</b>
+    </h2>
+
+    <section class="ml-2 mt-3 pr-4 pt-4 pl-4 pb-4 has-background-light has-border-radius">
```
Comment (tdruez) [inline vulnerabilities/templates/api_user_creation_form.html:25]: An additional paragraph, why one needs an API key and what that API will provide, would be nice.

**Review by tdruez — CHANGES_REQUESTED** [review]

File: vulnerabilities/templates/api_user_creation_form.html (hunk 2 above)
Comment (TG1999) [reply vulnerabilities/templates/api_user_creation_form.html:27]: Done!

**Pull request merged**